Library/include/CSP/Systems/Users/UserSystem.h

```diff
@@ -104,17 +107,13 @@ class CSP_API CSP_NO_DISPOSE UserSystem : public SystemBase
 	/// Note: The Authentication Provider and the Redirect URL you've passed in the first step will be used now
 	/// @param ThirdPartyToken csp::common::String : The authentication token returned by the Provider
 	/// @param ThirdPartyStateId csp::common::String : The state Id returned by the Provider
+	/// @param UserHasVerifiedAge csp::common::Optional<bool> : An optional bool to specify whether or not the user has verified that they are over 18
 	/// @param Callback LoginStateResultCallback : callback that contains the result of the Magnopus Connected Services Authentication operation
 	CSP_ASYNC_RESULT void LoginToThirdPartyAuthenticationProvider(const csp::common::String& ThirdPartyToken,
 																  const csp::common::String& ThirdPartyStateId,
+																  const csp::common::Optional<bool>& UserHasVerifiedAge,
 																  LoginStateResultCallback Callback);
 
-	/// @brief Log in to Magnopus Connected Services services as a guest.
-	/// @param DeviceId csp::common::String : The device Id to use when logging in
-	/// @param Callback LoginStateResultCallback : callback to call when a response is received
-	[[deprecated("This is no longer needed as we now have proper device ID generation! Please use LoginAsGuest() instead.")]] CSP_ASYNC_RESULT void
-		LoginAsGuestWithId(const csp::common::String& DeviceId, LoginStateResultCallback Callback);
```

Comment:
I chose to remove it here instead of updating this function to pass the new boolean flag. It has been deprecated for a long time now, and no clients seem to still use it, although the wrapper that the Unity team does use uses it. I can modify the merge commit to include the breaking change marker.